### PR TITLE
Add user-agent tracking, format few files.

### PR DESCRIPTION
**Pull request by pavithrasv**

### CODE REVIEW

**Comment by pavithrasv** [comment]
Fixes https://github.com/tensorflow/cloud/issues/113

File: src/python/tensorflow_cloud/core/deploy.py [hunk 1]
```diff
@@ -33,6 +35,9 @@
     VERSION = "2.1"
 
 
+_USER_AGENT_FOR_RUN_DEPLOY_TRACKING = "tf-cloud-run-deploy/" + version.__version__
```
[the remark this answers is not in the document]
Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/deploy.py:38]: We are using Google Cloud Build from run API as well. So i was thinking of possibly adding another one for cloud build call.

[the remark this answers is not in the document]
Comment (ucdmkt) [reply src/python/tensorflow_cloud/core/deploy.py:38]: As we discussed offline, we'll be able to distinguish it from separate endpoint.

[the remark this answers is not in the document]
Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/deploy.py:38]: Discussed offline. Changing this to a generic name.

File: src/python/tensorflow_cloud/core/deploy.py [hunk 2]
```diff
@@ -66,6 +71,7 @@ def deploy_job(
     job_id = _generate_job_id()
     project_id = gcp.get_project_name()
     ml_apis = discovery.build("ml", "v1", cache_discovery=False)
+    ml_apis._http = _set_user_agent(ml_apis._http)
```
[the remark this answers is not in the document]
Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/deploy.py]: Updated, thank you!

File: src/python/tensorflow_cloud/core/deploy.py [hunk 3]
```diff
@@ -232,3 +240,22 @@ def _generate_job_id():
...
     unique_tag = str(uuid.uuid4()).replace("-", "_")
     return "tf_cloud_train_{}".format(unique_tag)
+
+
+# TODO(pavithrasv): Move this and the discovery API stuff into common repo tools.
+class TFCloudHttpRequest(googleapiclient_http.HttpRequest):
+    """HttpRequest builder that sets a customized user-agent header for TF Cloud.
+
+  This is used to track the usage of the TF Cloud.
+  """
```
Comment (ucdmkt) [inline src/python/tensorflow_cloud/core/deploy.py:250]: Indentation looks inconsistent.

File: src/python/tensorflow_cloud/core/deploy.py [hunk 4]
```diff
@@ -16,14 +16,17 @@
 from __future__ import division
 from __future__ import print_function
 
+import httplib2
```
Comment (ucdmkt) [inline src/python/tensorflow_cloud/core/deploy.py:19]: this looks no longer used.

File: src/python/tensorflow_cloud/core/deploy.py [hunk 5]
```diff
@@ -232,3 +240,22 @@ def _generate_job_id():
...
+  This is used to track the usage of the TF Cloud.
+  """
+
+    def __init__(self, *args, **kwargs):
+        """Construct a HttpRequest.
+
+    Args:
+      *args: Positional arguments to pass to the base class constructor.
+      **kwargs: Keyword arguments to pass to the base class constructor.
+    """
```
Comment (ucdmkt) [inline src/python/tensorflow_cloud/core/deploy.py:258]: same as above. Is this intentional?

File: src/python/tensorflow_cloud/core/deploy.py (hunk 4 above)
Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/deploy.py]: Good catch!

**Pull request merged**